tests/providers/http/sensors/test_http.py

```diff
@@ -201,6 +201,9 @@ def prepare_request(self, request):
             self.response._content += ('/' + request.params['date']).encode('ascii', 'ignore')
         return self.response
 
+    def merge_environment_settings(self, _url, **kwargs):
+        return kwargs
+
```

Comment:
What's this for?

Reply:
This is test mock for `requests.Session.merge_environment_settings` code, which is called in https://github.com/apache/airflow/pull/14701/files#diff-b644517c333b1a654b85edd76607b296990c14f203b8521091458a46fa228359R184

It is needed to make tests work with `FakeSession` class instead of regular session.

Reply:
Oh this in test code! I missed that first time round :D